phpunit

```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-16 15:36 UTC
+// Cache-Id-Version: 2016-03-16 16:33 UTC
```

Comment:
I thought this is necessary every time we change the PHPUnit configuration file, isn't it?